library/src/amd_detail/rocblaslt/src/extops/CMakeLists.txt

```diff
@@ -34,6 +34,18 @@ list(REMOVE_DUPLICATES archs)
 set(python_venv_executable "${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME}")
 set(python_launch_prefix "PYTHONPATH=${PROJECT_BINARY_DIR}/lib" "${python_venv_executable}")
 set(dat_depends "")
+
+if($ENV{ENABLE_ADDRESS_SANITIZER})
```

Comment:
I don't have a lot of investment in these build files anymore, but ftr, this is not an appropriate use of an environment variable. I realize we probably do this for compatibility with something but in the future, these things should all be cache variables.